### ISSUE TITLE
Neurina killing chickens

### ISSUE THREAD

**Issue by ileleana** [TqLxQuanZ/DeceasedCraft#73]
Hello, I'm not sure what keeps happening on my server I am running the latest version of the pack with no changes but the Neurina mod keeps killing my chickens with the ticking entity fixer is this supposed to happen?

**Comment by TqLxQuanZ** [comment]
Yes, currently chicken has an issue when laying egg and it'll crash the server, so the mod neruina will kill them to prevent your server from crashing. It is a known issue and currently there's no work around for now.

**Comment by TqLxQuanZ** [comment]
Will be fixed in next version 5.4.2
